driver-mapping/src/test/java/com/datastax/driver/mapping/MapperCaseSensitivityTest.java

```diff
@@ -40,7 +40,7 @@
     static class User {
 
         @PartitionKey
-        @Column(caseSensitive = true)
+        @Column(name = "userId", caseSensitive = true)
```

Comment:
I like it!  Also fixes my other complaint about having to annotate everything with `@Column(caseSensitive=true)`.